english/wake-up/wake-up-again/dream2/dream2.md

```diff
@@ -3,5 +3,10 @@ You look around and you see big white cyclinders everywhere on all four sides.
 Suddenly an aircraft flies by. 
 In the shiny underbelly of the airplane you see a sea of white cyclinders stretched across the landscape. 
 You realize that you are just a marshmallow among thousands in a vast island of marshmallows.
-Another marshmallow takes you to the marshmallow chief, who explains to you that
-the dangers of sticks.
\ No newline at end of file
+Another marshmallow takes you to the marshmallow chief, who explains to you the dangers of sticks.
+
+Do You:
+
+[Listen to him, and settle into your dream living as a marshmallow](listen/listen.md)
+
+[Feel hungry. Marshmallow chief looks mighty tasty. Attempt marshicide and try to cannibalize Marshmallow chief](hungry/hungry.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.